### ISSUE TITLE
Cutaneous horn

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6555]
**Preferred term label:** Cutaneous horn

**Synonyms**
Cornu cutaneum

**Definition (free text, please give PubMed ID)**

A cutaneous horn is a keratinized non-malignant protrusion on the surface of the skin. Diagnosis is established by biopsy of the horn and  Biopsy of the horn and the underlying skin. PMID: 33085427

**Parent term (use hpo.jax.org/app)**
Localized skin lesion HP:0011355

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
